src/test/java/duke/DeadlineTest.java

```diff
@@ -0,0 +1,35 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+
+    @Test
+    public void testStringConversion() {
```

Comment:
you may want to name your exceptions more explicitly - indicate whether it is a success / expect an exception to be thrown.